acceptance/config_test.go

```diff
@@ -31,6 +31,10 @@ type TestConfig struct {
 	// If true, run this test when running with cloud env configured
 	Cloud *bool
 
+	// If true, this test is not run if -short is passed and cloud env is configured
+	// This implies Cloud = true.
+	CloudLong *bool
```

Comment:
How about:
```suggestion
	Expensive *bool
```

1. We already have a Cloud tag that can be composed with it.
2. Normally Long refers to a number, expensive will be more explicit

Reply:
We could have a separate flag for controlling short behavior, but here I'm specifically adding short scoped to cloud runs only. The reason is that all tests should be fast locally with mock server, so if a test works both as a local test and cloud test, it would only be slow on cloud.


"Long" is used because it's the opposite of "Short", "Expensive" does not have this property.

Reply:
Long is indeed confusing here, what about CloudSlow?

Reply:
CloudSlow is fine too, updated.